components/nimbus/build.rs

```diff
@@ -2,7 +2,50 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use std::process::Command;
+
 pub fn main() {
     #[cfg(feature = "uniffi-bindings")]
     uniffi_build::generate_scaffolding("./src/nimbus.udl").unwrap();
+
+    // Check if we have a Python virtual environment set up yet
+    let venv_exists = Command::new("[")
```

Comment:
Yeah, it is relying on `bash` syntax.  Let me break out a Windows machine and test it. (It may just mean it needs built in the linux subsystem for windows, maybe?)

Reply:
Yeah I wouldn't worry too much about it working for normal windows (WSL is what the windows developers on our team use IIRC) so doing the `ci full` check to make sure that taskcluster can build the windows artifacts is probably good enough

Reply:
yeah, I'm fine with ignoring windows in the shortish term. This does look slightly fragile though - what "python3" exactly? I think it might be slightly better to have a .py script doing some of this, so the .rs code has less python magic - "bootstrap_glean.py" with a `#!` line that specifies python3 and let it do this cruft, and the rust code just executes it via the shell? That even has a good chance of working on Windows if the "python launcher" is installed :) Regardless though, I don't think that need block this.